.github/workflows/c3-e2e-quarantine.yml

```diff
@@ -1,12 +1,8 @@
-# Runs c3 e2e tests for quarantined frameworks on merges to main
-
 name: C3 E2E (Quarantine)
 on:
-  push:
-    branches:
-      - main
-    paths:
-      - packages/create-cloudflare/**
+  schedule:
+    - cron: "0 3 * * *"
```

Comment:
I wonder if for quarantined tests we should fail if any of the tests actually pass and vice-versa. This would be a nice setup, since we would have to go and remove tests from quarantine when they start passing.
Then we can continue to run these in VP etc.